src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2184,6 +2184,9 @@ because xbuild doesn't support framework reference assemblies.
       Condition=" '%(_BuildTargetAbis.Identity)' == 'armeabi' "
       Text="Invalid value 'armeabi' in %24(AndroidSupportedAbis). This ABI is no longer supported. Please update your project properties to remove the old value. If the properties page does not show an 'armeabi' checkbox, un-check and re-check one of the other ABIs and save the changes."
   />
+  <AndroidError Code="XA1025"
+    ResourceName="XA1025"
+    Condition=" '%(_BuildTargetAbis.Identity)' == 'armeabi-v7a' AND '$(AndroidAotMode)' == 'Hybrid' " />
```

Comment:
As currently written, this new error will break builds for projects using `armeabi-v7a;arm64-v8a`, even though those projects would run successfully on arm64-v8a devices.

The idea is that even though users might currently be using `armeabi-v7a;arm64-v8a` "successfully," the inclusion of `armeabi-v7a` in `$(AndroidSupportedAbis)` gives a false impression that apps built with both ABIs enabled might work on armeabi-v7a devices too, when in fact they will not.

I think this might be an acceptable level of build breakage considering that the `Hybrid` setting is undocumented and therefore not expected to be used by many projects.